podcasts/MultiSelectHelper.swift

```diff
@@ -318,6 +321,23 @@ class MultiSelectHelper {
         actionDelegate.multiSelectActionCompleted()
     }
 
+    private class func share(actionDelegate: MultiSelectActionDelegate, view: UIView?) {
+        guard let episode = actionDelegate.multiSelectedBaseEpisodes().first as? Episode else {
+            return
+        }
+
+        Analytics.track(.podcastShared, properties: ["type": "episode", "source": "multi_select"])
+
+        if let view {
+            SharingHelper.shared.shareLinkTo(episode: episode, shareTime: 0, fromController: actionDelegate.multiSelectPresentingViewController(), sourceRect: view.bounds, sourceView: view)
+        }
+
+        // If no view is provided, the app is sharing from the bottom sheet
+        if let view = actionDelegate.multiSelectPresentingViewController().view {
+            SharingHelper.shared.shareLinkTo(episode: episode, shareTime: 0, fromController: actionDelegate.multiSelectPresentingViewController(), sourceRect: view.bounds, sourceView: view, showArrow: false)
+        }
```

Comment:
Both of the `SharingHelper` methods are called when I share from the overlay button. 

Wdyt about refactoring this to:

```suggestion
        guard let sourceView = view ?? actionDelegate.multiSelectPresentingViewController().view else {
            return
        }

        SharingHelper.shared.shareLinkTo(episode: episode,
                                         shareTime: 0,
                                         fromController: actionDelegate.multiSelectPresentingViewController(),
                                         sourceRect: sourceView.bounds,
                                         sourceView: sourceView,
                                         showArrow: view != nil)
```